next.config.js

```diff
@@ -48,6 +29,60 @@ module.exports = withPlugins([
           }
         }
       };
+
+      if (!isServer) {
```

Comment:
Makes sense! The intent here is to split out overlapping frontend code into chunks -- eliminate dupe code!

Reply:
Wild that this is `!isServer`, reminds me of meteor.js (back in the day!) ... code that runs on server and client (!!)